### ISSUE TITLE
Using statement removes CA2000 also on other IDisposable variables

### ISSUE THREAD

**Issue by Liander** [dotnet/roslyn-analyzers#3084]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

v2.9.8

#### Diagnostic ID

CA2000

#### Repro steps
Having an `using` statement without braces seems to remove IDisposable warning also for all other IDisposable variables. (An `using` with a braces-block does not cause this behaviour.)

``` csharp
class Program
{
   static void Main(string[] args)
   {
      var streamReader1 = new StreamReader("");
      var streamReader2 = new StreamReader("");
      
      // Removing the using below gives three CA2000-warnings, but having this one gives none!
      using var streamReader3 = new StreamReader(""); 
   }
}

```
#### Expected behavior
Two CA2000-warnings are expected in the above example.

#### Actual behavior
No warning given.

**Comment by mavasani** [comment]
This should be fixed if you move to latest version of Visual Studio: https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes
